matrix_client/client.py

```diff
@@ -171,6 +175,15 @@ def send_text(self, text):
     def send_emote(self, text):
         return self.client.api.send_emote(self.room_id, text)
 
+    def send_image(self,url,name,size,mimetype,width=500,height=500):
```

Comment:
We shouldn't have defaults for `width`/`height`. Better to omit non-required information than to give false information here.